action.yml

```diff
@@ -1,9 +1,9 @@
 name: 'Setup Julia environment'
 description: 'Setup a Julia environment and add it to the PATH'
 author: 'Sascha Mann'
-inputs: 
+inputs:
   version:
-    description: 'The Julia version to download (if necessary) and use. Example: 1.0.4'
+    description: 'The Julia version to download (if necessary) and use. Use a string input to avoid unwanted decimal conversion e.g. 1.10 will be interpreted as 1.1. Examples: "1", "1.10", "lts", "pre"'
```

Comment:
```suggestion
    description: 'The Julia version to download (if necessary) and use. Use a string input to avoid unwanted decimal conversion e.g. 1.10 without quotes will be interpreted as 1.1. Examples: "1", "1.10", "lts", "pre"'
```